docs/catalog/web/avoid-chaining-critical-requests.md

```diff
@@ -0,0 +1,41 @@
+---
+version: 1.0
+submitted_by: dubrie
+published_date: TBD
+category: web
+tags: 
+ - web
+ - role:web-developer
+ - size:medium
+---
+
+# Avoid Chaining Critical Requests
+
+## Description
+
+Most web experiences require a lot of work from the users browser. The intermediate steps between receiving the HTML, CSS, and JavaScript bytes and the request processing to turn them into rendered pixels is known as the Critical Rendering Path. [Critical request chains](https://web.dev/critical-rendering-path/) are a series of dependent network requests that block page rendering until they complete. The greater the length of the chains and the larger the download sizes, the more significant the impact on page load performance and the energy required to render a page.
+
+## Solution
+
+From an energy efficiency perspective, avoiding critical request chains reduces the overall energy required to render a page by using loading time more efficiency without a serialized set of requests and allowing the user to begin interacting with the content earlier, potentially negating the need for any further rendering and energy consumption. 
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Sofware Carbon Intensity Spec](https://github.com/Green-Software-Foundation/software_carbon_intensity)
```

Comment:
Should this be changed to the shortened `https://grnsft.org/sci` link as described in https://github.com/Green-Software-Foundation/green-software-patterns/issues/94 ? 🤔

Reply:
Fixed!